src/seedu/addressbook/AddressBook.java

```diff
@@ -377,6 +388,61 @@ public static String executeCommand(String userInputString) {
     private static String getMessageForInvalidCommandInput(String userCommand, String correctUsageInfo) {
         return String.format(MESSAGE_INVALID_COMMAND_FORMAT, userCommand, correctUsageInfo);
     }
+    
+    /**
+     * List all current persons in alphabetical order.
+     */
+    
+    private static String executeSortPersons(){
+        ArrayList<String[]> toBeDisplayed = new ArrayList<String[]>(getAllPersonsInAddressBook());
+        		
+        Collections.sort(toBeDisplayed, new Comparator<String[]>(){
+                                        	public int compare(String[] personA, String[] personB){
+                                        		return getNameFromPerson(personA).compareToIgnoreCase(getNameFromPerson(personB));
+                                        	}
+                                        }
+        );
+        showToUser(toBeDisplayed);
+        return getMessageForPersonsDisplayedSummary(toBeDisplayed);
+    }
+    
+    /**
+     * Edits a person's properties
+     * Deletes the person and adds him again with the new properties
+     * 
+     * User suppose to list all persons first to specify index of person to edit.
+     * User keys in existing person's index, then will be prompted to enter new informations in the same format as add.
+     * 
+     * @param commandArgs
+     * @return
+     */
+    
+    private static String executeEditPerson(String commandArgs){
+        executeDeletePerson(commandArgs);
+        
+        String newProperties = "dummy " + getNewProperties();
+        
+        String[] propertyArray = splitCommandWordAndArgs(newProperties);
+        //String commandType = commandTypeAndParams[0];
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.